packages/components/src/Button/Button.stories.tsx

```diff
@@ -23,21 +23,23 @@ export default {
   },
   decorators: [
     // Workaround Storybook's wrapping flex parent that make everything full width
-    story => <div>{story()}</div>,
+    Story => (
+      <div>
+        <Story />
+      </div>
+    ),
   ],
-} as ComponentMeta<typeof Button>;
+} as Meta<typeof Button>;
 
-const BasicTemplate: ComponentStory<typeof Button> = args => (
-  <Button {...args} />
-);
-
-export const Basic = BasicTemplate.bind({});
-Basic.args = {
-  label: "New Job",
-  onClick: () => alert("👍"),
+export const Basic: StoryObj<typeof Button> = {
+  args: {
+    label: "New Job",
+    type: "primary",
+    onClick: () => alert("👍"),
```

Comment:
This is what I meant by using "actions", instead of `alert()`:

```suggestion
import { fn } from "storybook/test";

    onClick: fn(),
```